pages/_en-US/twilightmenu/custom-dsi-3ds-sfx.md

```diff
@@ -39,13 +40,12 @@ If your file is in Stereo, you should also go to **Tracks > Mix > Mix Stereo dow
 
 You must set `PlayStartupJingle=1` in your `theme.ini` for the startup jingle to play.
 
-
 ## Menu BGM
 Menu BGM needs to be a **16-bit 16 kHz Mono** raw PCM file. Below are two methods for converting audio files into that format.
 
-Unlike sfx.bin, *bgm.pcm.raw* can be arbitrarily large.
+Unlike sfx.bin, there is no size limit to `bgm.pcm.raw`.
 
-### ffmpeg
+### ffmpeg (Terminal)
 The simplest way to convert music for use in TWiLight Menu++ is to run this [ffmpeg](https://ffmpeg.org) command in a terminal:
 
 ```bash
```

Comment:
I can't make a suggestion because it's not changed, but while you're at it you should probably change this to use WAV instead of raw PCM, it's just better. I'd need to look up the settings though.

Ditto on the Audacity section.

Reply:
Could you look up the settings for me so I could reflect them?

Reply:
ffmpeg:
```bash
ffmpeg -i [input file] -acodec adpcm_ima_wav bgm.wav
```

Audacity:
```md
1. Load the file in Audacity
1. Select `File` > `Export` > `Export as WAV`
1. Set `Encoding` to `IMA ADPCM`
1. Set the output name to `bgm.wav` and click `Save`
1. Click `OK` to the metadata editing
```

I just tried with a 44100 Hz stereo song and both ways it seems to work perfectly to me. ffmpeg has like a couple dozen slightly different ADPCMs so not sure if this is the actual correct one but it sounds right to my non-musically inclined ears.